src/plot.js

```diff
@@ -452,3 +488,81 @@ class FacetMap2 extends FacetMap {
     return this;
   }
 }
+
+// The Grid class is defined here so we can prepend it per the grid option
+// see grid.js for the helpers
+const defaults = {
+  ariaLabel: "grid",
+  fill: "none",
+  stroke: "currentColor",
+  strokeOpacity: 0.1
```

Comment:
Aside: something I ran into with the Delaunay mesh mark, but I kind of wanted `Plot.grid({stroke: "red"})` to reset the stroke opacity to 1.0 rather than using the default 0.1. I.e., if the stroke is undefined, then the default stroke is _currentColor_ and the default stroke opacity is 0.1, but if the stroke is defined, then the default stroke opacity resets to 1.0.

Reply:
Yes I also struggled with this — it’s not obvious visually if it’s a stroke width or opacity that makes the line faint, and you have to either inspect or refer to the documentation. I like the suggested change.